### DISCUSSION TITLE
Gibt es ein Zulassungsverfahren für Anbieter eines DCC-Validierungsservices?

### DISCUSSION THREAD

**Discussion by raphaelm** [corona-warn-app/cwa-website#3285] [Q&A]
Die FAQ auf der CWA-Website gibt an:

> Gibt es ein Zulassungsverfahren für Anbieter eines DCC-Validierungsservices?
> 
> Interessierte Unternehmen können ein Prüfverfahren beim BMG durchlaufen. Das Verfahren ist für alle Anbieter offen.

Weitere Informationen hierzu sind weder auf coronawarn.app, noch auf der Website der europäischen Kommission, noch auf der Website des BMG, noch auf der Website des RKI zu finden. In den FAQ steht weiter oben was von ISO27001 und was von Datenschutz, aber mehr nicht.

Als neugieriger Bürger frage ich mich: Was umfasst dieses Prüfverfahren?

Als hypothetisch interessiertes Unternehmen frage ich mich weiterhin: Welche Abteilung im BMG ist dafür zuständig und wie erreicht man diese?

Wäre toll, wenn die FAQ hier ergänzt werden könnten oder sich hier auf GitHub eine Antwort findet.

**Comment by MikeMcC399** [comment]
@raphaelm 

- Eine ähnliche Frage stellte ich im https://github.com/corona-warn-app/cwa-website/issues/2218.

Eine vollständige Antwort ist noch offen. Es gab gestern ein Update zum Blog [Corona-Warn-App vereinfacht die Zertifikatsprüfungen bei der Ticketbuchung](https://www.coronawarn.app/de/blog/2021-12-20-cwa-2-15/).

**Comment by raphaelm** [comment]
Da steht 

> Bereits seit Oktober 2021 liegen die europaweit gültigen Anforderungen an Onlineverifikationsdienste vor. 

Auf der EU-Seite finden sich auch alle technischen Anforderungen in der Spezifikation, aber nichts zu organisatorischen Anforderungen und Antragstellung :(

**Comment by Ein-Tim** [comment]
@raphaelm Alle Anbieter eines DCC-Validierungsservices haben sich zurückgezogen und das Feature wurde auf Eis gelegt. Kann dieser Issue geschlossen werden? Siehe auch https://github.com/corona-warn-app/cwa-website/issues/2218#issuecomment-1094809124.

**Comment by raphaelm** [comment]
Naja, klar, hat jetzt deutlich an Relevanz verloren, aber falls das Thema zB im Herbst doch nochmal relevant wird wäre ja doch *zumindest* der Ansprechpartner interessant, wenn schon nicht der Kriterienkatalog. Aber klar, ist jetzt nicht mehr wirklich dringend ;)

**Comment by raphaelm** [comment]
Hier ist das Thema auch noch offen: https://fragdenstaat.de/anfrage/zulassung-von-dcc-validierungsservices/

**Reply by brianebeling** [reply]
Bis zum 19. Februar 2022 kann diese Anfrage noch beantwortet werden. Ich fürchte, da das Feature aktuell eingestellt wurde und die Frist noch vor Herbst liegt, dass es dort wahrscheinlich keine Antwort mehr geben wird.

**Comment by MikeMcC399** [comment]
This question is now obsolete due to the ramp-down of the app and infrastructure.
